doc/guide/guide-bloch.rst

```diff
@@ -29,7 +29,7 @@ Before getting into the details of these objects, we can simply plot the blank B
 .. plot::
     :context:
 
-    b.make_sphere()
+    b.render()
```

Comment:
Why change this?
Both work.

Reply:
Because of

> We have `render` and `make_sphere` methods that do exactly the same (`make_sphere` calls render) I would vote to deprecate `make_sphere` for QuTiP 5.0.

from issue #1913